task/08-objects-tasks.js

```diff
@@ -23,7 +23,13 @@
  *    console.log(r.getArea());   // => 200
  */
 function Rectangle(width, height) {
-    throw new Error('Not implemented');
+    this.width = width;
+    this.height = height;
+
+    Rectangle.prototype.getArea = () => width * height;
```

Comment:
It's better to move this line outside Rectangle function, because everytime function Rectange be invoked - every time prototype will be changed

Reply:
I need to fix that?

Reply:
Yes, just move this line below function Rectangle (now it is not so significant , but in a big real project it may cause the worse performance).

Reply:
> Yes, just move this line below function Rectangle (now it is not so significant , but in a big real project it may cause the worse performance).

If i change this line, my third test not be success :c